### PR TITLE
Solution: Support manual job triggering

### PR DESCRIPTION
**Pull request by ericdude4**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -3,7 +3,7 @@ defmodule Quantum.Mixfile do
 
   use Mix.Project
 
-  @version "3.2.0"
+  @version "3.3.0"
```
Comment (maennchen) [inline mix.exs:6]: Please remove that line sice the release will be a separate commit.

File: lib/quantum/job_broadcaster.ex [hunk 2]
```diff
@@ -300,6 +300,33 @@ defmodule Quantum.JobBroadcaster do
...
+        "[#{inspect(Node.self())}][#{__MODULE__}] Running job #{inspect(name)} once"
+      end)
+
+    case Map.fetch(jobs, name) do
+      :error ->
+        {:noreply, [], state}
+
+      {:ok, job} ->
+        # Send event to telemetry incase the end user wants to monitor events
+        :telemetry.execute([:quantum, :job, :run], %{}, %{
```
Comment (maennchen) [inline lib/quantum/job_broadcaster.ex:321]: Telemetry events for execution are done here: https://github.com/quantum-elixir/quantum-core/blob/master/lib/quantum/executor.ex

I think this event is not needed, what do you think?

**Review by maennchen — CHANGES_REQUESTED** [review]
Thanks a lot for the PR. There’s two small things that need some attention. I‘ll try it out in a project manually tomorrow.

File: mix.exs (hunk 1 above)
Comment (ericdude4) [reply mix.exs]: Done :)

File: lib/quantum/job_broadcaster.ex (hunk 2 above)
Comment (ericdude4) [reply lib/quantum/job_broadcaster.ex]: If the telemetry event gets handled in the executor, then I agree it's not necessary here. I have made the adjustment in my next commit :)

**Review by maennchen — APPROVED** [review]

**Comment by maennchen** [comment]
@ericdude4 Works great, I'll prepare a release :)

**Comment by maennchen** [comment]
Released as `v3.3.0`
